internal/cmd/backup.go

```diff
@@ -116,8 +127,10 @@ func backupCmdFunc(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("error closing backup encoder: %w", err)
 	}
 
-	if err := f.Sync(); err != nil {
-		return fmt.Errorf("error syncing backup file: %w", err)
+	if f != os.Stdout {
```

Comment:
Shouldn't sync still work on os.Stdout? It's still a *File.

Reply:
You can call it, but it will return an error.

Reply:
If we can tell it's an `inappropriate ioctl for device` error, we could just specifically ignore that error instead.

Reply:
<img width="482" alt="Screenshot 2023-08-23 at 1 12 06 PM" src="https://github.com/authzed/zed/assets/343539/179af8a5-c4d6-4653-9e05-11bd0c0d46ae">
Looks like there are multiple types of errors. I think this code is reasonable to avoid trying to catch a million error cases (especially if they differ cross platform).